src/applications/edu-benefits/0994/submitForm.js

```diff
@@ -0,0 +1,50 @@
+import _ from 'lodash';
+import { submitToUrl } from 'platform/forms-system/src/js/actions';
+
+import {
+  transformForSubmit,
+  recordEvent,
+} from 'platform/forms-system/src/js/helpers';
+
+const submitForm = (form, formConfig) => {
+  const body = formConfig.transformForSubmit
+    ? formConfig.transformForSubmit(formConfig, form)
+    : transformForSubmit(formConfig, form);
+
+  return submitToUrl(
+    body,
+    formConfig.submitUrl,
+    formConfig.trackingPrefix,
+  ).then(results => {
+    recordEvent({
+      event: `${formConfig.trackingPrefix}-submission-successful`,
```

Comment:
Per my other comment, you can remove this line so you don't trigger the event twice.